web/src/components/jobs/JobDetailPage.tsx

```diff
@@ -207,48 +214,70 @@ const JobDetailPage: FunctionComponent<IProps> = (props) => {
         </Box>
       </Box>
       <Grid container spacing={2}>
-        <Grid item xs={4}>
+        <Grid item xs={3}>
           <MqInfo
             icon={<CalendarIcon color={'disabled'} />}
             label={'Created at'.toUpperCase()}
             value={formatUpdatedAt(job.createdAt)}
           />
         </Grid>
-        <Grid item xs={4}>
+        <Grid item xs={3}>
           <MqInfo
             icon={<CalendarIcon color={'disabled'} />}
             label={'Updated at'.toUpperCase()}
             value={formatUpdatedAt(job.updatedAt)}
           />
         </Grid>
-        <Grid item xs={4}>
+        <Grid item xs={3}>
           <MqInfo
-            icon={<SpeedRounded color={'disabled'} />}
+            icon={<Speed color={'disabled'} />}
             label={'Last Runtime'.toUpperCase()}
             value={job.latestRun ? stopWatchDuration(job.latestRun.durationMs) : 'N/A'}
           />
         </Grid>
-        <Grid item xs={4}>
+        <Grid item xs={3}>
+          <MqInfo
+            icon={<Title color={'disabled'} />}
+            label={'Type'.toUpperCase()}
+            value={job.type ? job.type : 'N/A'}
+          />
+        </Grid>
+        <Grid item xs={3}>
           <MqInfo
             icon={<Start color={'disabled'} />}
             label={'Last Started'.toUpperCase()}
             value={job.latestRun ? formatUpdatedAt(job.latestRun.startedAt) : 'N/A'}
           />
         </Grid>
-        <Grid item xs={4}>
+        <Grid item xs={3}>
           <MqInfo
             icon={<SportsScore color={'disabled'} />}
             label={'Last Finished'.toUpperCase()}
             value={job.latestRun ? formatUpdatedAt(job.latestRun.endedAt) : 'N/A'}
           />
         </Grid>
-        <Grid item xs={4}>
+        <Grid item xs={3}>
           <MqInfo
             icon={<DirectionsRun color={'disabled'} />}
             label={'Running Status'.toUpperCase()}
             value={<MqStatus label={job.latestRun?.state} color={jobRunsStatus(latestRuns)} />}
           />
         </Grid>
+        <Grid item xs={3}>
+          <MqInfo
+            icon={<EscalatorWarning color={'disabled'} />}
+            label={'Parent Job'.toUpperCase()}
+            value={
+              job.parentJobName ? (
+                <MQTooltip title={job.parentJobName}>
+                  <>{truncateText(job.parentJobName, 16)}</>
+                </MQTooltip>
+              ) : (
+                'None'
```

Comment:
_Minor_: I'd prefer we standardize on `N/A` (which is used in other pages as well)